frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -12,11 +12,26 @@ import { addIDPItems } from './oauth';
 import { TextFilter } from '../factory';
 import { fuzzyCaseInsensitive } from '../factory/table-filters';
 import { withExtensions, isGlobalConfig, GlobalConfig } from '@console/plugin-sdk';
+import i18next from 'i18next';
 
 const stateToProps = (state: RootState) => ({
   configResources: state.k8s.getIn(['RESOURCES', 'configResources']),
 });
 
+export const breadcrumbsForGlobalConfig = (
+  detailsPageKind: string,
+  detailsPageUrl: string,
+) => () => [
```

Comment:
Why does this return a function instead of just returning the breadcrumbs array?

Reply:
Currently DetailsPage only has a [breadcrumbsFor](https://github.com/openshift/console/blob/1c9b925b8d722e9781a60c11fe39258d90bf916a/frontend/public/components/factory/details.tsx#L102) prop., which is a function.  Breadcrumbs component does take a `breadcrumb` prop which is a straight array of breadcrumb objs.  I'll add it to DetailsPageProps.

Reply:
Hi @spadgett, updated